Dockerfile

```diff
@@ -10,26 +10,23 @@
 # Stage 1: build the plugin
 # use a 'fat' image to setup the dependencies we'll need
 
-FROM python:3.10 AS builder
+FROM python:3.12 AS builder
```

Comment:
With the tox env being moved to `py3`, should this move with it to `python:3`? Or would that create too many potential differences between runs?

Reply:
Great suggestion. I'll leave it for now as-is in the PR, but I'll give the suggestion a place in my long-term-thoughts-collection. Maybe for a future update?